content/en/dynamic_instrumentation/_index.md

```diff
@@ -37,24 +37,23 @@ Dynamic Instrumentation requires the following:
 
 - [Datadog Agent][1] 7.41.1 or higher is installed alongside your service.
 - [Remote Configuration][2] is enabled in that Agent.
-- For Java applications, tracing library [`dd-trace-java`][3] 1.5 or higher.
-- For Python applications, tracing library [`dd-trace-py`][4] 1.5 or higher.
-- For .NET applications, tracing library [`dd-trace-dotnet`][5] 2.22 or higher.
+- For Java applications, tracing library [`dd-trace-java`][3] 1.8.0 or higher.
```

Comment:
I don't understand what you mean 🙂 I pushed another commit yesterday with version updates.

All versions are set to latest, I think that's fine since they include all recent bugfixes. We will need to update the python version once we release all the updates we are working on now.